pkg/kubelet/kubelet_pods.go

```diff
@@ -264,9 +264,10 @@ func (kl *Kubelet) GeneratePodHostNameAndDomain(pod *v1.Pod) (string, string, er
 }
 
 // GetPodCgroupParent gets pod cgroup parent from container manager.
-func (kl *Kubelet) GetPodCgroupParent(pod *v1.Pod) (cm.CgroupName, string) {
+func (kl *Kubelet) GetPodCgroupParent(pod *v1.Pod) (string, string) {
```

Comment:
rather than fix the merge cycle introduced in the rebase this way, can we just have GetPodCgroupParent return a single string result for the concrete value?  the first return result is not used anywhere else i *think*